bin/compile_search_parser.sh

```diff
@@ -7,4 +7,4 @@ npx pegjs -o src/lib/headline_filter_parser.{js,grammar.pegjs}
 # we're telling eslint to ignore the generated file.
 PARSER_FILE=src/lib/headline_filter_parser.js
 # Prepend the generated parser code with a `eslint-disable` comment.
-echo "/* eslint-disable */"|cat - $PARSER_FILE > /tmp/out && mv /tmp/out $PARSER_FILE
+sed -i '1i \/* eslint-disable */' "$PARSER_FILE"
```

Comment:
Good refactoring. I didn't use awk or sed, because I wanted to keep the script as platform agnostic as possible. I used only bash features, because bash is already a dependency of running the script.

But yeah, with adding bash as a dependency it will only run on a Unix based system anyway and that is likely to have sed just as well.

Whenever the first Windows contributor comes along, she can refactor it, I guess(;

Having thought more on it, good call on sed. It's definitively cleaner code and it's likely running on the same systems as bash.

Reply:
I think, even if we would relay on a POSIX shell, not bash, a POSIX sed would be available.

Reply:
Works for me!